plugins/com.gwtplugins.gdt.eclipse.apiclientlib/.classpath

```diff
@@ -1,38 +1,39 @@
-<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
This might be an EOL change, or a spacing change, but they must be avoided, or done in an ad-hoc commit, as they make reviewing way more complicate

Reply:
Click the "gear" icon in the diff view, and then select "hide whitespace", or just add ?w=1 to the url